ui/intermittent-failures/BugDetailsView.jsx

```diff
@@ -137,17 +137,29 @@ const BugDetailsView = props => {
         <React.Fragment>
           <Row>
             <Col xs="12" className="text-left">
-            <Breadcrumb>
+              <Breadcrumb>
                 <BreadcrumbItem>
-                  <Link title="Intermittent Failures View" to={'/'}>Treeherder</Link>
+                  <a
+                    title="Treeherder home page"
+                    href="https://treeherder.mozilla.org/"
```

Comment:
I think you use `/#/` here, that way if you're working locally it'll route to `localhost:5000//treeherder.mozilla.org/`.

Reply:
@sarah-clements I have replaced "https://treeherder.mozilla.org/" with "/#/" for Home's href, hope that is what you mean't. Requesting your review.